.github/workflows/release.yml

```diff
@@ -1,47 +1,153 @@
 name: Release
 
-on: [workflow_dispatch]
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        required: true
+      publish_doc:
+        description: 'If set to true, then the documentation will be published'
+        default: 'false'
+        required: false
 
 jobs:
-  build_test_release:
-    runs-on: ubuntu-18.04
+  build_and_test_ubuntu:
+    strategy:
+     matrix:
+        platform: [ubuntu-18.04]
+        mgversion:
+          - 1.4.0
+    runs-on: ${{ matrix.platform }}
     steps:
-    - name: Install system dependencies
+    - name: Install system dependencies (Ubuntu 18.04)
       run: |
         sudo apt install -y libpython3.7 python3-pip python3-setuptools
+        python3 -m pip install --upgrade
         sudo pip3 install networkx pytest pyopenssl sphinx
         mkdir ~/memgraph
-        curl -L https://memgraph.com/download/memgraph/v1.2.0/ubuntu-18.04/memgraph_1.2.0-community-1_amd64.deb > ~/memgraph/memgraph_1.2.0-community-1_amd64.deb
+        curl -L https://download.memgraph.com/memgraph/v${{matrix.mgversion}}/ubuntu-18.04/memgraph_${{matrix.mgversion}}-community-1_amd64.deb > ~/memgraph/memgraph-community.deb
         sudo ln -s /dev/null /etc/systemd/system/memgraph.service # Prevents Memgraph from starting.
-        sudo dpkg -i ~/memgraph/memgraph_1.2.0-community-1_amd64.deb
-
-    - name: Install mgclient
-      uses: actions/checkout@v2
-      with:
-        repository: memgraph/mgclient
-        ref: 'v1.1.0'
-    - name: Install mgclient
-      run: |
-        mkdir build
-        cd build
-        cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF ..
-        make
-        sudo make install
+        sudo dpkg -i ~/memgraph/memgraph-community.deb
```

Comment:
cache it maybe like in mgclient the docker image?
https://github.com/memgraph/mgclient/blob/de2c0e5bb41207a59028aec79f137f01c2979c0c/.github/workflows/ci.yml#L53